src/Apps/Artist/Routes/AuctionResults/Components/EmptyArtistAuctionResults.tsx

```diff
@@ -0,0 +1,32 @@
+import { Flex, Text, Button, GridColumns, Column } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+
+export const EmptyArtistAuctionResults: React.FC = () => {
+  return (
+    <Flex>
+      <GridColumns width="100%" gridRowGap={2}>
+        <Column span={6} start={4}>
+          <Text variant="md" textAlign="center">
+            There are no auction results for this artist at this time.
+          </Text>
+        </Column>
+        <Column span={6} start={4}>
+          <Text variant="md" color="black60" textAlign="center">
+            The page will be updated once auction results are added. In the
+            meantime, check out the millions of auction results and art market
+            data.
+          </Text>
+        </Column>
+        <Column span={12} alignItems="center">
+          <Flex width="100%" justifyContent="center">
+            <RouterLink to="/price-database">
+              <Button variant="secondaryNeutral">
```

Comment:
We would want to use `as={RouterLink} to="/price-database"` on the `Button` instead of wrapping it; this currently creates two tab-stops; one of which doesn't do anything. It will complain about a type error: it's safe to `// @ts-ignore` it